### ISSUE TITLE
[Feature Request] Create balance adjustment transaction with Reconciliation

### ISSUE THREAD

**Issue by kennydrobnack** [actualbudget/actual#57]
I'm just getting started really using Actual and have found one feature that is present in other software is absent in Actual. Not a major feature but it would be a big time saver in a workflow that I do on a few accounts once a month.

It has reconciliation features, but no way to automatically create an adjustment transaction. Example: my Actual budget says one of my accounts has $500 but my bank balance says $450. If I enter my bank balance is $450 when I start reconciling, Actual currently will tell me how off I am from my bank balance.

In other software, there is the option to click a button to create the adjustment transaction.

The message in Actual looks like this: "Your cleared balance ACTUAL_BALANCE needs ADJUSTMENT_AMOUNT to match
your bank's balance of BANK_BALANCE" and there's a "Done Reconciling" button to close the window.

There should be a "Create adjustment transaction" button that will create the adjustment transaction.

Discussion: ideally if you're on top of your finances, you'd figure out where the $50 difference is, and enter the transactions for the missing amount. However, I have off-budget accounts for my investments and on the last day of the month, go into these accounts and see what the current balance is. Having to manually create the balance adjustment transactions slows this workflow down quite a bit.

With other software I can just go check my current account balances, click "Reconcile", paste in the current amount and then click the "Create adjustment transaction" button and be done with that account.

**Comment by TomAFrench** [comment]
Agreed, this is something that I sorely miss from YNAB4 for similar reasons (tracking investment accounts)
